src/components/extensive/DemographicsCollapseGrid/types.ts

```diff
@@ -5,7 +5,7 @@ import { Framework, useFrameworkContext } from "@/context/framework.provider";
 import { DemographicCollections } from "@/generated/v3/entityService/entityServiceConstants";
 import { DemographicDto, DemographicEntryDto } from "@/generated/v3/entityService/entityServiceSchemas";
 
-export type DemographicEntity = "projectReports" | "siteReports";
+export type DemographicEntity = "projectReports" | "siteReports" | "sites" | "projects";
```

Comment:
I'm curious about this change. `sites` don't have demographics. `projects` do, but also this change doesn't seem to have anything to do with this ticket.